src/content/operationDashboard/DashboardView.tsx

```diff
@@ -9,6 +9,7 @@ import {
   FrameOptions,
 } from "amazon-quicksight-embedding-sdk";
 import StyledSelect from "../../components/StyledFormComponents/StyledSelect";
```

Comment:
There are some weird overflowing corners after you click the dropdown then click away to close the dropdown and the input is still in focus.

![Screenshot 2024-11-08 at 9 45 16 AM](https://github.com/user-attachments/assets/005c698f-2afd-4c59-9121-52f4f7a2d5f1)

I think I found the cause, this seems to fix it. Could you add something like this directly to the StyledSelect component?

```javascript
"&.Mui-focused .MuiOutlinedInput-input:focus": {
    borderRadius: "8px",
},
```

Reply:
That's some really odd behavior. Thanks for catching it, fixed in b8cd159.